.avalanche-cli.json

```diff
@@ -1,5 +1,6 @@
 {
   "node-config": {
     "log-level": "info"
-  }
-}
+  },
+  "singlenodeenabled": false
```

Comment:
[nit] needs hyphenation like other values

Reply:
this name is interpreter by avalanche cli, we can't control it